mkosi/config.py

```diff
@@ -557,6 +557,36 @@ def parse_path(
     return path
 
 
+def parse_paths_and_dirs(
```

Comment:
This does not seem appropriately named anymore. `parse_paths_from_directory()` maybe?